runners/spark/src/test/java/org/apache/beam/runners/spark/TfIdfTest.java

```diff
@@ -39,6 +51,194 @@
  */
 public class TfIdfTest {
 
+  /**
+   * A transform containing a basic TF-IDF pipeline. The input consists of KV objects
+   * where the key is the document's URI and the value is a piece
+   * of the document's content. The output is mapping from terms to
+   * scores for each document URI.
+   */
+  public static class ComputeTfIdf
```

Comment:
To break the dependency, just wipe this test out. It is redundant with `examples/complete/TfIdfTest.java`. In order to have an integration test for `TfIdf`, you should create `TfIdfIT` and then all runners can run it if they like. But I don't think that has to happen in this PR.